### PR TITLE
Keys: add clipboard warning :speech_balloon:

### LINKED ISSUE #1550: Add warning in wizard before copying the seed to clipboard
While translating I came across:

<location filename="../wizard/WizardMemoTextInput.qml" line="65"/>
<source>Seed copied to clipboard</source>

I did not come across a warning however, informing the user about the potential security risk for putting the seed in plain text into the clipboard.

Please consider adding a warning like
"Coyping your seed to clipboard exposes you to the threat of malicious software recording your seed and sharing it with third parties. To be sure please write your seed down manually."

### PR DESCRIPTION
**Pull request by mmbyday**
Fixes #1550
seed

### CODE REVIEW

File: pages/Keys.qml [hunk 1]
```diff
@@ -115,14 +115,53 @@ Rectangle {
...
+                        Layout.topMargin: 10 * scaleRatio
+                        source: "../images/warning.png"
+                    }
+
+                    Text {
+                        id: warningTextSeed
+                        Layout.topMargin: 12 * scaleRatio
+                        Layout.preferredWidth: statusRect.width - 80 * scaleRatio
+                        Layout.leftMargin: 6 * scaleRatio
+                        text: qsTr("WARNING: Copying your seed to clipboard can expose you to malicious software which may record your seed then steal your monero. Please write your seed down manually.") + translationManager.emptyString
```
Comment (selsta) [inline pages/Keys.qml]: `WARNING: Copying your seed to clipboard can expose you to malicious software, which may record your seed and steal your Monero. Please write down your seed manually.`

I asked on IRC and got this suggestion from luigi :)

**Comment by xmrdsc** [comment]
This does not fix #1550 btw.

File: pages/Keys.qml (hunk 1 above)
Comment (mmbyday) [reply pages/Keys.qml]: Cool, good improvement. Thanks for gathering suggestions @selsta . Thanks @luigi1111  :) :) Updated.

**Comment by mmbyday** [comment]
What else should happen, @xmrdc ?

**Comment by xmrdsc** [comment]
@mmbyday #1550 refers to the wizard, this PR changes the Keys page, so #1550 should stay open. 

Nice PR though, reviewed.

**Comment by Leza89** [comment]
> This does not fix #1550 btw.

Nice Catch

**Pull request merged**